### PR TITLE
Force isvlan tag on uint16 types

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

**Comment by stcheng** [comment]
is this used to have logics for checking if a vlanid is valid or not?

**Comment by kcudnik** [comment]
It will be used for that purpose

File: inc/saiacl.h [hunk 1]
```diff
@@ -1192,6 +1194,7 @@ typedef enum _sai_acl_entry_attr_t
      *
      * @type sai_acl_field_data_t sai_uint16_t
      * @flags CREATE_AND_SET
+     * @isvlan false
```
Comment (lguohan) [inline inc/saiacl.h]: can we make isvlan=false by default so that we do not need to mark this for all uint16?

Comment (kcudnik) [inline inc/saiacl.h]: it is false by default, but i made it explicit on uint16 types for future adding attributes for users to not forget to set it to true if attribute is vlan

**Comment by kcudnik** [comment]
Is there any other issue with this PR ?

**Pull request merged**